homeassistant/components/glances/sensor.py

```diff
@@ -172,6 +172,11 @@ async def async_update(self):
                     if sensor["type"] == "temperature_core":
                         if sensor["label"] == self._sensor_name_prefix:
                             self._state = sensor["value"]
+            elif self.type == "temperature_hdd":
+                for sensor in value["sensors"]:
+                    if sensor["type"] == "temperature_hdd":
+                        if sensor["label"] == self._sensor_name_prefix:
+                            self._state = sensor["value"]
```

Comment:
```suggestion
                    if sensor["type"] == "temperature_hdd" and sensor["label"] == self._sensor_name_prefix:
                        self._state = sensor["value"]
```

Reply:
Thank you for the suggestion

Reply:
This line of code is too long and fails the check. The pre-commit hook suggests more lines of changes like this:

```
black....................................................................Failed
- hook id: black
- files were modified by this hook
pre-commit hook(s) made changes.
If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
To run `pre-commit` as part of git workflow, use `pre-commit install`.
All changes made by hooks:
diff --git a/homeassistant/components/glances/sensor.py b/homeassistant/components/glances/sensor.py
index a575e6cd..5a920879 100644
--- a/homeassistant/components/glances/sensor.py
+++ b/homeassistant/components/glances/sensor.py
@@ -174,7 +174,10 @@ class GlancesSensor(Entity):
                             self._state = sensor["value"]
             elif self.type == "temperature_hdd":
                 for sensor in value["sensors"]:
-                    if sensor["type"] == "temperature_hdd" and sensor["label"] == self._sensor_name_prefix:
+                    if (
+                        sensor["type"] == "temperature_hdd"
+                        and sensor["label"] == self._sensor_name_prefix
+                    ):
                         self._state = sensor["value"]
             elif self.type == "memory_use_percent":
                 self._state = value["mem"]["percent"]
Error: Process completed with exit code 1.
```

In this case, I think the original code is shorter and change it back

Reply:
Please address this comment. Flat is better than nested.

Reply:
This caused more lines than the original code, but anyway, I can change it.

Reply:
It's updated in cf3aed2